.github/workflows/post-merge-messages.yml

```diff
@@ -16,6 +16,19 @@ jobs:
       github.event.pull_request.user.login != 'lucwhite'
     runs-on: ubuntu-latest
     steps:
+      - name: Get Changed Files
+        id: changed_files
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
Since the fetch-depth here is 2, I'm wondering if we might need `persist-credentials: true`
And since the PR could be created from a fork, we may need `repository: ${{ github.event.pull_request.head.repo.full_name }}`

Reply:
hmm, or maybe we won't need to set repository since the commit is already on the base

Reply:
ok yeah, we might not need the credentials either then 😅